README.md

```diff
@@ -1,21 +1,100 @@
 # Heimdall Enterprise Server 2.0
 
+![Run E2E Backend + Frontend Tests](https://github.com/mitre/heimdall2/workflows/Run%20E2E%20Backend%20+%20Frontend%20Tests/badge.svg) ![Run Frontend Tests](https://github.com/mitre/heimdall2/workflows/Run%20Frontend%20Tests/badge.svg) ![Run Backend Tests](https://github.com/mitre/heimdall2/workflows/Run%20Backend%20Tests/badge.svg)
+
 This repository contains the source code for the Heimdall 2 Backend and Frontend (Heimdall-Lite).
 
-## How to Install
+### Demos
```

Comment:
This should have a `# Heimdall Lite` heading so that readers understand the below does not represent the Enterprise Server.

Reply:
Fixed, this now has a `Heimdall Lite` heading.